oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/conditions/Checks.java

```diff
@@ -0,0 +1,118 @@
+package org.apache.jackrabbit.oak.commons.conditions;
```

Comment:
missing license header.

Reply:
Ups; had a conflict due to line ends, and that was lost.